### ISSUE TITLE
[macos][v2.12.4]route: illegal option -- e

### ISSUE THREAD

**Issue by rprimus** [AirVPN/airvpn-client#57]
Sat Apr 15 16:38:25 BST 2017

```src/Platforms.Osx/Osx.cs:246``` (where RouteList is defined), uses the same route command as used in ```src/PlatformsLinux/Linux.cs```:
```
250:			string result = ShellCmd("route -n -ee");
```

From the logfile (when Generate  System Report is called):

```
31959:. 2017.04.15 16:33:10 - Shell of 'sh','-c 'route -n -ee'' done sync in 29 ms, Output: route: illegal option -- e
```

For MacOS,  the command should be  ```netstat -rnl```.  This produces output with 8 columns (not the 10 columns that the current function expects).

**Comment by Clodo76** [comment]
Will be fixed in the next release. Anyway, this part of code that read routes exists only for generating a .txt report, nothing really used. It's used under Linux and Windows, to remove default route gateway (opt-in, generally useless) and probably removed in future.

**Comment by rprimus** [comment]
Fri Jun  9 16:57:43 BST 2017

Fixed in ```2.13.1```.
